src/app/lib/utilities/preprocessor/rules/addIdsToItems.test.js

```diff
@@ -0,0 +1,98 @@
+import addIdsToItems from './addIdsToItems';
+
+jest.mock('uuid', () => () => 'mockId');
+
+const noId = {
```

Comment:
```js
const noId = {
  content: {
    groups: [
      {
        items: [
          {
            headlines: 'Test headlines',
          },
          {
            name: 'This is a test headline',
          },
        ],
      },
    ],
  },
}
```
What do think about removing `attributes` that we don't need to test `addIdsToItems`.

Reply:
👍 Makes sense to remove un-needed attributes